src/status_im2/contexts/onboarding/enter_seed_phrase/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.contexts.onboarding.enter-seed-phrase.view
+  (:require [quo2.core :as quo]
+            [quo.core :as quo1]
```

Comment:
why quo1? (quo also has text input)

Reply:
not sure if covers use case, but this is one: https://github.com/status-im/status-mobile/tree/develop/src/quo2/components/inputs/input

Reply:
in  quo2, it just called input, instead of text-input ([api](https://github.com/status-im/status-mobile/blob/45da51bea6fefd0d392b4b6834e8587022174b07/src/quo2/core.cljs#L146))